pkg/printers/internalversion/printers_test.go

```diff
@@ -664,7 +664,7 @@ func TestPrintNodeOSImage(t *testing.T) {
 					Addresses: []api.NodeAddress{{Type: api.NodeExternalIP, Address: "1.1.1.1"}},
 				},
 			},
-			// Columns: Name, Status, Roles, Age, KubeletVersion, NodeInternalIP, NodeExternalIP, OSImage, KernelVersion, ContainerRuntimeVersion
+			// Columns: Name, Status, Roles, Age, KubeletVersion, NodeInternalIP, NodeExternalIP, OSImage, KernelVersion (Arch), ContainerRuntimeVersion
```

Comment:
I don't think it is really necessary to update the comments here, especially since the test has not been changed to include architecture in the test case output.

I think it would be fine to just leave all these columns comments as-is (for this and all similar comments below).